### PR TITLE
updated readme, added init container for database setup

### PR DESCRIPTION
**Pull request by matthiashub-porsche**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,228 +1,225 @@
...
+| `CX_AUTH_ISSUER_URL`      | URL    | `http://localhost:8081/auth/realms/catenax` | IAM token issuer url |
+| `CX_AUTH_REALM`           | String | `catenax` | IAM realm |
+| `CX_AUTH_ROLE_MAPPINGS`   | String | `create_wallets:add_wallets,view_wallets:view_wallets,update_wallets:update_wallets,delete_wallets:delete_wallets,view_wallet:view_wallet,update_wallet:update_wallet` | IAM role mapping |
+| `CX_AUTH_RESOURCE_ID`     | String | `ManagedIdentityWallets` | IAM resource id |
+| `CX_AUTH_CLIENT_ID`       | String | `ManagedIdentityWallets` | IAM client id |
+| `CX_AUTH_CLIENT_SECRET`   | String | `ManagedIdentityWallets-Secret`| It can be extracted from keycloak under **realms** &gt; catenax &gt; clients &gt; ManagedIdentityWallets &gt; credentials |
+| `APP_VERSION`             | String | `0.4.0` | application version, this should be in-line with  |
+| `ACAPY_API_ADMIN_URL`     | String | `http://localhost:11000` | admin url of ACA-Py |
+| `ACAPY_ADMIN_API_KEY`     | String | `Hj23iQUsstG!dde` | admin api key of ACA-Py enpoints |
+| `ACAPY_LEDGER_URL`        | String | `https://indy-test.idu.network/register` | Hyperledger Indy ledger url for registration |
```
Comment (jpbu) [inline README.md:62]: I think this property is a leftover from an earlier workaround and no longer used

File: README.md [hunk 2]
```diff
@@ -1,228 +1,225 @@
...
 
-### Option 1: Run from source <a id= "startupDockerContainers"></a>
+1. Role `update_wallet` requires the BPN of Caller and it can be used:
+    * to issue Verifiable Credentials (The BPN of issuer will be checked)
+    * to issue Verifiable Presentations (The BPN of holder will be checked)
+    * to store Verifiable Credentials (The BPN of holder will be checked)
+    * to trigger Business Partner Data update for its own BPN
 
-Starting up Docker Containers for Postgres, Keycloak and AcaPy via following steps:
+Additionaly a Token mapper can to be created under *Clients > ManagedIdentityWallets*
```
Comment (jpbu) [inline README.md:220]: Additionaly -> Additionally

File: README.md [hunk 3]
```diff
@@ -234,187 +231,209 @@ Add to access token : ON
...
-  * Run the script generateDidFromSeed script with `./generateDidFromSeed.sh` which starts the AcaPy container and shows the printout of the DID and VerKey from its logs in the console like the following
+  * Run the script generateDidFromSeed script with `./generateDidFromSeed.sh` which starts the
+    ACA-Py container and shows the printout of the `DID` and `VerKey` from its logs in the console
+    like the following
     ```
-    2022-08-12 08:08:13,888 indy.did DEBUG get_my_did_with_meta: <<< res: '{"did":"Hw2eFhr3KcZw5JcRW45KNc","verkey":"AEErMofs7DcJT636pocN2RiEHgTLoF4Mpj6heFXwtb3q","tempVerkey":null,"metadata":null}'
+    2022-08-12 08:08:13,888 indy.did DEBUG get_my_did_with_meta: <<< res: '{"did":"HW2eFhr3KcZw5JcRW45KNc","verkey":"aEErMofs7DcJT636pocN2RiEHgTLoF4Mpj6heFXwtb3q","tempVerkey":null,"metadata":null}'
     ```
-  * If the script did not stop the container, the command `docker-compose down -v` can stop and delete it manually
+  * If the script did not stop the container, the command `docker compose down -v` can stop and delete it manually
```
Comment (jpbu) [inline README.md:395]: I believe `docker-compose` (with a hyphen) is the correct command (there are multiple occasions with `docker compose`)

**Review by jpbu — COMMENTED** [review]
I only had a look at the changes and rendered README, looks good with a few (non-critical) comments. I did not verify the changes for the local setup procedure since it requires rebuilding it from scratch - regarding the "init container", is it safe that this doesn't break when executing a second time (e.g. when the container is rebuilt)?

File: README.md (hunk 3 above)
Comment (matthiashub-porsche) [reply README.md:395]: I understood from https://docs.docker.com/compose/ that in newer Docker versions it is `docker compose`, on my system both commands work.

Comment (jpbu) [reply README.md:395]: Thanks for the information, I wasn't aware of that, but still think we should be consistent and not use a mix of `docker-compose` and `docker compose` in the README and scripts. We should also add a short note on the prerequisites if we require a certain version of Docker and/or Docker Compose.

**Pull request merged**